CMakeModules/BuildElixir.cmake

```diff
@@ -0,0 +1,62 @@
+##
+## Copyright (c) 2018-2019 Fred Dushin <fred@dushin.net>, Riccardo Binetti <rbino@gmx.com>
```

Comment:
No need for Fred Dushin listed here.

Reply:
Ok, I left it since it was "heavily derived" from your work